lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -86,7 +86,7 @@ def install!
                 end
               end
 
-              if target.requires_frameworks?
+              if target.build_as_dynamic_framework?
```

Comment:
same for this one

Reply:
This is the old logic, but may need to change to eliminate warnings with the new build system.